### DISCUSSION TITLE
settings GPS NMEAvsUBLOX inav7 Beintian B880/B880Q

### DISCUSSION THREAD

**Discussion by petrweigl** [iNavFlight/inav#9663] [Q&A] [answered]
When I want to use a GPS module in iNav 7.0, which I have previously used with the NMEA protocol (recently purchased BN880 and BN880Q), is it sufficient to just switch to the UBX protocol in the iNav configurator? Or do I also need to use the UBLOX u-center software for some configuration changes directly in the GPS module? Thanks for explanation

**Comment by sensei-hacker** [comment, marked as answer]
You can just switch the protocol in Configurator.
INAV tells the GPS "I'd like to speak Ublox, please".
